### PR TITLE
Use internal Pbench server to send results to.

### PR DESCRIPTION
**Pull request by jmencak**
This PR allows pbench-copy/move-results work out of the box on ansible-host.

### CODE REVIEW

File: roles/pbench_storage/vars/main.yml [hunk 1]
```diff
@@ -5,3 +5,7 @@ pbench_storage_device: '/dev/vda'
 pbench_storage_device_partition: 3
 # Pbench agent default directory.
 pbench_storage_mount_path: '/var/lib/pbench-agent'
+# Pbench results redirector host.
+pbench_results_redirector: 'pbench.perf.lab.eng.bos.redhat.com'
```
Comment (mbruzek) [inline roles/pbench_storage/vars/main.yml]: Could this be read from an environment variable?

Comment (jmencak) [reply roles/pbench_storage/vars/main.yml]: It could if we need it.

File: roles/pbench_storage/templates/pbench-agent.cfg.j2 [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+[results]
+user = pbench
+host_path = http://%(pbench_result_redirector)s/pbench-archive-host
+ssh_opts = -o StrictHostKeyChecking=no
+scp_opts = -o StrictHostKeyChecking=no
+webserver = %(pbench_web_server)s
+host_info_url = http://%(webserver)s/pbench-results-host-info.versioned/pbench-results-host-info.URL001
+dir = /pbench/public_html/incoming
+
+[pbench/tools]
```
Comment (jeremyeder) [inline roles/pbench_storage/templates/pbench-agent.cfg.j2]: Is this the correct values for the intervals? @chaitanyaenr

**Comment by chaitanyaenr** [comment]
@jmencak we can set this while building images. I see that it has been set to pbench.dev.openshift.com for the latest images. We have added support in svt repo to push the results to one of the internal servers like the keys e.t.c so we can change the default server to one of the internal pbench servers. 

Also I think we shouldn't maintain a template for config as it might change in the future. We need to use the one packaged in the rpm's.

File: roles/pbench_storage/templates/pbench-agent.cfg.j2 (hunk 2 above)
Comment (chaitanyaenr) [reply roles/pbench_storage/templates/pbench-agent.cfg.j2]: We set it to 10 while registering the tools.

**Comment by jmencak** [comment]
@chaitanyaenr completely agree on the subject of the maintainance of the template file, that was on my mind also when I was creating this.  Let's have this set correctly at qcow2 build process then.  Closing PR.

**Pull request closed without merging**